src/systems/ec/implementations/react/storybook/.storybook/manager.js

```diff
@@ -12,6 +12,7 @@ const theme = create({
 });
 
 addons.setConfig({
+  showRoots: false,
```

Comment:
We are currently using the default which is true. Why changing?

Reply:
Ah, this is a mistake, yeah, i moved this according to the docs for storybook 6 but i changed it's value inadvertently, it was set to true.